.github/workflows/ci.yaml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ghc: ['8.10', '8.8', '8.6']
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2.3.1
+    - name: Setup haskell
+      uses: actions/setup-haskell@v1.1.1
+      with:
+        ghc-version: ${{ matrix.ghc }}
+    - name: Build euphoria
```

Comment:
Can you build dependencies in a separate step? `cabal v2-build --dep` should work.

Reply:
Done

Reply:
I'll do the same for the tests

Reply:
You can cabal v2-build --dep --enable-tests --enable-benchmarks when building dependencies. This is just to make it easier to understand why CI fails.

Reply:
https://github.com/tsurucapital/euphoria/pull/26/commits/821ef82735c07456586a068dd9569fc340a7e277